code/cellular_automaton/src/von_neumann_cellular_automata/README.md

```diff
@@ -1,7 +1,37 @@
 # cosmos
 > Your personal library of every algorithm and data structure code that you will ever encounter
 
-# Cellular Automaton
+# Von Neumann Cellular Automaton
+
+Jonh Von Neumann worked for years on a simple model for a logical cellular automaton with the purpose to build a self-replication machine. Wich logical organization an automaton will need to be able to control itself in a way that it reproduces itself? To answer that, Von Neumann thought on logical operations and states with different objectives, capable to construct and deconstruct other states for their own. This lead to Von Neumann Cellular System.
+
+## States
+
+To maintain your simplicity, Von Neumann limited your automaton to only 29 states, is that the minimum to build a self-replication machine. This states can be divided into four groups:
+
+ - The **blank state** or unexcitable state (**U**) is the basic and initial state, when it is excited it will become one of the transmission states or a confluent state. If another state has been deconstructed, that cell returns to the blank state.
```

Comment:
```suggestion
 - The **blank state** or unexcitable state (**U**) is the basic and initial state; when it is excited it will become one of the transmission states or a confluent state. If another state has been deconstructed, that cell returns to the blank state.
```